src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractMethodOrPropertyOrEventSymbolReferenceFinder.cs

```diff
@@ -18,52 +18,6 @@ protected AbstractMethodOrPropertyOrEventSymbolReferenceFinder()
         {
         }
 
-        protected override async Task<ImmutableArray<(ISymbol symbol, FindReferencesCascadeDirection cascadeDirection)>> DetermineCascadedSymbolsAsync(
```

Comment:
this is the logic that moved up to the SymbolSet sublcasses.  The unidirectional sybmol set only has to deal with the .Down cases in this code.  The bidirectional set has to deal with both the up and down cases.